package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "loklak-php",
+  "version": "0.0.1",
+  "description": "Loklak PHP Dependencies",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/loklak/loklak-php.git"
+  },
+  "keywords": [
+    "loklak",
+    "php"
+  ],
+  "author": "Sudheesh Singanamalla <sudheesh95@gmail.com> (http://www.sudheesh.info/)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/loklak/loklak-php/issues"
```

Comment:
@sudheesh001 Please change the URL to https://github.com/loklak/loklak_api_php/issues
Even though it is redirecting to the right page